### PR TITLE
Tools: add a way to install sample backstage in plain kubernetes

### PR DESCRIPTION
**Pull request by eloycoto**

### CODE REVIEW

File: hack/manifests/backstage/Readme.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Install parados on plain kubernetes
+
+
+## Install kind
```
Comment (gabriel-farache) [inline hack/manifests/backstage/Readme.md:4]: this title is confusing: I expected steps to install `kind`  (or a link to https://kind.sigs.k8s.io/docs/user/quick-start/) but instead, it is the command to initialize the local cluster with kind

File: hack/manifests/backstage/Readme.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Install parados on plain kubernetes
```
Comment (pkliczewski) [inline hack/manifests/backstage/Readme.md:1]: what about having make target for this?

Comment (eloycoto) [reply hack/manifests/backstage/Readme.md:1]: It changed a bit, it was a draft PR, look at it now. I do not think that makes sense, but yeah.

File: hack/manifests/backstage/postgres.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: apps/v1
+kind: Deployment
```
Comment (RichardW98) [inline hack/manifests/backstage/postgres.yaml:3]: my understanding is that most of the time, db should be statefulset for persist storage.  in the future our backend need to use an external db too for production(h2 should only be for local testing)

File: hack/manifests/backstage/app-config.yaml [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    # Default Helmet Content-Security-Policy values can be removed by setting the key to false
+  cors:
+    origin: http://localhost:8080
+    methods: [GET, HEAD, PATCH, POST, PUT, DELETE]
+    credentials: true
+
+  database:
+    client: pg
+    connection:
+      host: ${POSTGRES_SERVICE_HOST}
```
Comment (RichardW98) [inline hack/manifests/backstage/app-config.yaml:32]: postgres.default.svc.cluster.local

Comment (eloycoto) [reply hack/manifests/backstage/app-config.yaml:32]: nope, that means the namespace is the default, using this way will inherit the right one.

File: hack/manifests/backstage/postgres.yaml (hunk 3 above)
Comment (RichardW98) [reply hack/manifests/backstage/postgres.yaml:3]: yes let's keep it in deploy

**Review by RichardW98 — APPROVED** [review]

**Pull request merged**